examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,105 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are pulled from an H5 file and are fit to extract the cross relaxivity of the sample. The R1(p) is first interpolated by fitting the self relaxivity to a polynomial. This fit is then used to fit the linear regime of the enhancement data. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+
+target_directory = os.path.normpath(getDATADIR("AG_processed_data"))
+h5_folder = "ras.h5"
+filename = "220616_E37"
+# {{{Dataset Parameters
+SL_conc_M = 455.2223e-6
+ppt = 1.5167e-3
+Ep_pts = 18
+# }}}
+# {{{ plotting fn
+def list_symbs(f):
+    # {{{ this is just to show all the parameters
+    list_symbs = []
+    for j, k in f.output().items():
+        s_repr = latex(Symbol(j))
+        list_symbs.append(f"${s_repr} = {k:0.5g}$")
+    list_symbs = "\n".join(list_symbs)
+    # }}}
+    return list_symbs
+
+
+# }}}
+with figlist_var() as fl:
+    # {{{ load data
+    Ep = nddata_hdf5(f"{h5_folder}/{filename}/Ep", directory=target_directory)
+    T1p = nddata_hdf5(f"{h5_folder}/{filename}/T1p", directory=target_directory)
```

Comment:
Why not stored as R1?

Reply:
This question should be for the script that makes the table of integrals. At this point in processing this is what we have.

Reply:
changed the processing script that generates this particular dataset to save R1p instead but note this will need to be done for all future post processing prior to running this script

Reply:
You should write a short script that can adapt old format for this and the other issue you noted (I noted you could write a short script using h5py

Reply:
you still need to write this script

Reply:
the other issue that I note is discussed [here](https://jmfrancklab.slack.com/archives/CLMMYDD98/p1697637883567909?thread_ts=1697637620.788759&cid=CLMMYDD98)